packages/bpk-component-price-range/src/BpkPriceRange.tsx

```diff
@@ -0,0 +1,160 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useEffect, useRef, useState } from 'react';
+
+import clamp from 'lodash/clamp';
+
+import BpkText, { TEXT_STYLES } from '../../bpk-component-text/src/BpkText';
+import { cssModules } from '../../bpk-react-utils';
+
+import BpkPriceMarker from './BpkPriceMarker';
+import { MARKER_TYPES } from './common-types';
+
+import type { MarkerType } from './common-types';
+
+import STYLES from './BpkPriceRange.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+type Marker = {
+  price: string;
+  percentage: number;
+};
+
+export type Props = {
+  min?: number;
+  max?: number;
+  showPriceIndicator?: boolean;
+  marker: Marker;
+  segments: {
+    low: Marker;
+    high: Marker;
+  };
+};
+
+const BpkPriceRange = ({
```

Comment:
usually we should allow the pass trough rest props to make the component expandable, eg
```
const BpkPriceRange = ({
  marker,
  max = 100,
  min = 0,
  segments,
  showPriceIndicator = false,
  ...rest
}
// ....
```

Reply:
Just some worries that any type of props could be passed, and included `classnames`

Reply:
This is correct, we want this to keep restricted and not allow overriding of properties so this would be acceptable here to not include it.

If we were wrapping a 3rd party library or using other Backpack utility components then I would say yes add `...rest` but for this component its not required